### PR TITLE
added initial genesis key descriptions

### PR DESCRIPTION
**Pull request by kevinhammond**
Added initial genesis parameter documentation

### CODE REVIEW

**Review by Jimbo4350 — COMMENTED** [review]
We should consolidate  `doc/reference/genesis-file-format.md` and `doc/reference/shelley-genesis.md` with `doc/getting-started/understanding-config-files.md`.`

**Comment by kevinhammond** [comment]
I think the correct approach is to prune down the getting started section and refer from it to the reference documentation (which is where I would expect to find detailed documentation) - once we add in all the Plutus parameters, it would overwhelm this section

**Review by olgahryniuk — APPROVED** [review]

File: doc/reference/genesis-file-format.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Genesis File Entries
+
+The possible genesis entries are described here.
+
+## Byron-specific Entries
+
+TODO: document these!
```
Comment (newhoggy) [inline doc/reference/genesis-file-format.md:7]: There are some TODOs here.

File: doc/reference/genesis-file-format.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Shelley-specific Entries
+
+These entries are also used for the Allegra and Mary eras.
+TODO: document these!
```
Comment (newhoggy) [inline doc/reference/genesis-file-format.md:12]: And here.

File: doc/reference/genesis-file-format.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+|  `maxTxExUnits` |     `{ "exUnitsMem": <nat>, "exUnitsSteps" : <nat> }`| Optional  |  Maximum number of Plutus memory units/CPU steps allowed for a single transaction. |
+|  `maxBlockExUnits` |     `{ "exUnitsMem": <nat>, "exUnitsSteps" : <nat> }`| Optional  |  Maximum number of Plutus memory units/CPU steps allowed for a complete block. |
+| `maxValueSize`  | `<nat>`  | Optional |  Maximum size of a Plutus `Value` in a transaction output. |
+| `collateralPercentage`  | `<nat>`  | Optional | The percentage of Plutus script execution fee that must be provided as collateral. |
+| `maxCollateralInputs`  |  `<nat>` | Optional |  The maximum number of collateral inputs that can be provided in a transaction. |
+| `costModels`  | `<costmodel>`  | Optional |  Plutus cost models (see below). |
+
+### Plutus Cost Models
+
+TODO: Provide this.
```
Comment (newhoggy) [inline doc/reference/genesis-file-format.md:31]: And here.

**Comment by newhoggy** [comment]
This PR is very old and hasn't made progress.

Please recreate if there is still intent to make this change.

**Pull request closed without merging**